README.md

```diff
@@ -1,18 +1,24 @@
 ## TDR Generated GraphQL
 
-This project generates, packages and deploys to the sonatype nexus the case classes generated by the [sbt-graphql](https://github.com/muuki88/sbt-graphql) project.
+This project generates, packages and deploys the case classes generated by the [sbt-graphql](https://github.com/muuki88/sbt-graphql) project to the sonatype nexus.
 
-These classes will be used by other TDR repositories to communicate with the consignment API.
+These classes are be used by other TDR repositories to communicate with the consignment API.
 
 ### Building locally
-* Add a new query file to `src/main/graphql`
+* If you add a new query file to `src/main/graphql`:
 * Add a schema file generated by the [consignment api](https://github.com/nationalarchives/tdr-consignment-api) project. This will be generated in the `target/sbt-graphql` directory and should be copied into `src/main/resources/schema.graphql`
```

Comment:
Why have you put `If you add` here?

Reply:
I don't fully remember exactly why I did, I think it sounds better without it anyway so will change it back 👌